internal/db/migrations/010000_create_initial_tables.up.sql

```diff
@@ -53,3 +47,28 @@ CREATE TABLE products (
 ENGINE=InnoDB
 DEFAULT CHARSET=utf8mb4
 COMMENT='A service or product that caring markets for its b2b customers';
+
+
+CREATE TABLE calls (
+    call_id         BIGINT NOT NULL PRIMARY KEY,
+    sid             BIGINT,
```

Comment:
We should add a comment to this field which describes what it is.

Reply:
So you can actually make the comments on the column with the `COMMENT` keyword. that way anyone viewing the DB from a console can see them.

Reply:
Ohh ok haha, will do that thank you